scripts/update/Updater.php

```diff
@@ -507,6 +507,30 @@ public function update($initialVersion) {
         }
         $this->skip('3.6.6', '3.6.13');
 
+        if ($this->isVersion('3.6.13')) {
+
+            // create new column for test center and delivery
+            $persistenceId = $this->getServiceManager()->get(DeliveryMonitoringService::CONFIG_ID)->getOption(DeliveryMonitoringService::OPTION_PERSISTENCE);
+            $persistence = \common_persistence_Manager::getPersistence($persistenceId);
+            $schemaManager = $persistence->getDriver()->getSchemaManager();
+            $schema = $schemaManager->createSchema();
+            $fromSchema = clone $schema;
+            try {
+                $tableLog = $schema->getTable(DeliveryMonitoringService::TABLE_NAME);
+                $tableLog->addColumn(DeliveryMonitoringService::COLUMN_DELIVERY_ID, "string", array("notnull" => true, "length" => 255));
```

Comment:
Column value cannot be null and this column will not be added to the table with existing data. Update script will fail with `column "delivery_id" contains null values` message.
Add default value to this column and then drop it:
```
$tableLog->addColumn(DeliveryMonitoringService::COLUMN_DELIVERY_ID, "string", array("notnull" => true, "length" => 255, 'default' => ''));
```
After column will be created call
```
ALTER TABLE delivery_monitoring ALTER COLUMN delivery_id DROP DEFAULT;
```
The same thing should be done for test center column